bench/README.md

```diff
@@ -38,6 +38,7 @@ Other Scalameta configurations progressively enable more and more features:
 Benchmark                      Mode    Cnt Score
 QuickScalacBaseline            sample  35  1595.274 ± 15.263  ms/op
 QuickScalacRangepos            sample  35  1703.097 ± 25.044  ms/op  (6% overhead)
+QuickScalametaOpt              sample  15  1934.902 ± 29.232  ms/op  (21% overhead)
```

Comment:
Wow, is that a speedup from 42% to 21% overhead?

Reply:
It's just 25% to 21% :)